local-volume/provisioner/pkg/common/common.go

```diff
@@ -99,6 +103,12 @@ type LocalPVConfig struct {
 	AffinityAnn     string
 }
 
+// ProvisionerConfiguration defines Provisioner configuration objects
+type ProvisionerConfiguration struct {
```

Comment:
Add a comment here that each field needs to be individually marshalled.  And a TODO to see if there's a better way.

Reply:
done